docker/env.sh

```diff
@@ -2,11 +2,11 @@
 # Set environment variables here.
 
 export NS3_HOME="/opt/carma-simulation/bin/fed/ns3"
-export JAVA_HOME="/usr/lib/jvm/java-11-jdk-amd64"
+export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
 export SUMO_HOME="/opt/sumo/sumo-1_15_0"
 export CARLA_HOME="/opt/carla"
 export CARMA_SIMULATION_HOME="/opt/carma-simulation"
 export PATH="/usr/local/share/sumo/bin:/usr/local/share/sumo/tools:$PATH"
```

Comment:
I suspect this SUMO_HOME is used for the carla-mosiac bridge for sumo traci client py files. Sumo is downloaded to opt/sumo/ but cmake presumably installs it to /usr/local/ although I am not sure that includes the python sumo client.

Reply:
Yeah traci python API is included under sumo/tools which I do not think is built or installed by CMake. See guide for including sumo traci API in python (https://sumo.dlr.de/docs/TraCI/Interfacing_TraCI_from_Python.html)

Reply:
We need SUMO either way since CDA Sim launches SUMO. I assume this is just a way of making sure we get the TraCI client that corresponds to our SUMO installation.